src/utils/styles.js

```diff
@@ -1,16 +1,11 @@
 import autoPrefix from '../styles/auto-prefix';
-import update from 'react-addons-update';
 import warning from 'warning';
 
 const reTranslate = /((^|\s)translate(3d|X)?\()(\-?[\d]+)/;
 
 const reSkew = /((^|\s)skew(x|y)?\()\s*(\-?[\d]+)(deg|rad|grad)(,\s*(\-?[\d]+)(deg|rad|grad))?/;
 
-function mergeSingle(objA, objB) {
-  if (!objA) return objB;
-  if (!objB) return objA;
-  return update(objA, {$merge: objB});
-}
+export const mergeStyles = (...args) => Object.assign({}, ...args);
```

Comment:
> I'm recommending we just reimplement the components to use the prepareStyles from context directly, not from the style-propable mixin, and anywhere we were calling mergeStyles() we replace with Object.assign. After that this can go away.

Oh, right. So all the more reason to ignore what I said :D :D This is great as it is :grin: